content/reference/v1beta1/application.md

```diff
@@ -184,19 +184,13 @@ The minimum KOTS version that is required by the release. Introduced in KOTS v1.
 
 Specifying this in the application spec of the release will enforce compatibility checks for both new installations and application updates, and will block the installation or update if the currently deployed KOTS version is less than the specified minimum KOTS version.
 
-### Limitations
-
-`minKotsVersion` is not supported in the following cases:
-
-* Channels that have [semantic versioning](/vendor/packaging/promoting-releases/#semantic-versioning) enabled.
-* The minimum version increases and then decreases from one release to the next.
+### Guidance for informing users of the need to update
 
-`minKotsVersion` is not supported in these cases because when the minimum version increases, the admin console can error when retrieving it (described [below](#guidance-for-informing-users-of-the-need-to-update)). If the minimum KOTS version then decreases in the next release, KOTS will successfully retrieve that release, which means the intermediate release is lost.
+After promoting a new release that specifies a minimum KOTS version that is greater than what an end user currently has deployed, that application version appears in the version history of the admin console after checking for updates, but the application version is not downloaded. The admin console temporarily displays an error message inline with this pending version, informing the end user that they must update KOTS in order to download this application version. This error is also displayed when the end user checks for updates with the [`kots upstream upgrade` command](/kots-cli/upstream/).
```

Comment:
```suggestion
After promoting a new release that specifies a minimum KOTS version that is greater than what an end user currently has deployed, that application version appears in the version history of the admin console after checking for updates, but the application version is not downloaded. The admin console temporarily displays an error message inline with this pending download version, informing the end user that they must update KOTS in order to download this application version. This error is also displayed when the end user checks for updates with the [`kots upstream upgrade` command](/kots-cli/upstream/upgrade/).
```

Reply:
You make a good point, though I agree that I'm not sure "pending download version" is perfect. Maybe I'll say, "The admin console temporarily displays an error message inline with this new version."